modulefiles/noaacloud.lua

```diff
@@ -2,15 +2,19 @@ help([[
 Load environment to build UPP on NOAA Cloud
 ]])
 
-
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/upp-addon-env/install/modulefiles/Core")
-
-stack_intel_ver=os.getenv("stack_intel_ver") or "2021.3.0"
+prepend_path("MODULEPATH", "/contrib/spack-stack-rocky8/spack-stack-1.6.0/envs/upp-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/apps/modules/modulefiles")
+prepend_path("PATH", "/contrib/EPIC/bin")
+load("gnu")
+load("stack-intel")
+load("stack-intel-oneapi-mpi")
```

Comment:
These loads should not be necessary as they are run below with versions specified.

Reply:
@weihuang-jedi Have you verified successful initial testing on CSPs with these updates?

Reply:
@FernandoAndrade-NOAA I have compiled on AWS with Rocky 8, but did not run others.

-- Installing: /contrib/Wei.Huang/dev/upp-cloud/install/lib/cmake/upp/upp-targets.cmake
-- Installing: /contrib/Wei.Huang/dev/upp-cloud/install/lib/cmake/upp/upp-targets-release.cmake
+ rm -rf /contrib/Wei.Huang/dev/upp-cloud/exec
+ mkdir -p /contrib/Wei.Huang/dev/upp-cloud/exec
+ cp ../install/bin/upp.x /contrib/Wei.Huang/dev/upp-cloud/exec/.

Do you have suggestions to run some tests here?

Thanks,

Wei

Reply:
UPP standalone RTs are currently not configured on CSPs, so a build test would be sufficient for this PR. I tested on AWS with Rocky 8 with no errors during the build, I will work on verifying Azure tomorrow.